barcode/site_sections/barcode.py

```diff
@@ -0,0 +1,12 @@
+from barcode import *
+from barcode_utils import generate_barcode_from_badge_num, get_badge_num_from_barcode
+
+# TODO: restrict to just admins for this stuff, probably.
```

Comment:
If you require people access, you are already restricting it to admin access, so you can remove this TODO.